content/library/advanced-features/dataframes.md

```diff
@@ -228,298 +224,88 @@ st.data_editor({
 
-### Column configuration
+## Configuring columns
 
-When working with data in Streamlit, the [`st.column_config`](/library/api-reference/data/st.column_config) class is a powerful tool for configuring data display and interaction. Specifically designed for the `column_config` parameter in [`st.dataframe`](/library/api-reference/data/st.dataframe) and [`st.data_editor`](/library/api-reference/data/st.data_editor), it provides a suite of methods to tailor your columns to various data types - from simple text and numbers to lists, URLs, images, and more.
+You can configure the display and editing behavior of columns via `st.dataframe` and `st.data_editor` via the [Column configuration API](/library/api-reference/data/st.column_config). We have developed the API to let you add images, charts, and clickable URLs in dataframe and data editor columns. Additionally, you can make individual columns editable, set columns as categorical and specify which options they can take, hide the index of the dataframe, and much more.
 
-Whether it's translating temporal data into user-friendly formats or utilizing charts and progress bars for clearer data visualization, column configuration not only provides the user with an enriched data viewing experience but also ensures that you're equipped with the tools to present and interact with your data, just the way you want it.
+Column configuration includes Text, Number, Checkbox, Selectbox, Date, Time, Datetime, List, Link, Image, Line chart, Bar chart, and Progress bar columns. There is also a generic Column option. See the embedded app below to view these different column types. Each column type is individually previewed in the [Column configuration API](/library/api-reference/data/st.column_config) documentation.
 
-<TileContainer>
-<RefCard href="/library/api-reference/data/st.column_config/st.column_config.column">
-<Image pure alt="screenshot" src="/images/api/column_config.column.jpg" />
+<Cloud src="https://doc-column-config-overview.streamlit.app/?embed=true&embed_options=disable_scrolling" height="480"/>
 
-#### Column
+### Format values
 
-Configure a generic column.
+You can specify the format of your values in two ways: through column configuration or through [`pandas.DataFrame.style`](https://pandas.pydata.org/docs/reference/style.html). When using `st.data_editor`, formatting for editable columns must be declared through column configuration and not a pandas `Styler` object. Only disabled columns in a data editor can accept formatting from a pandas `Styler` object.
```

Comment:
I don't think we should push people towards pandas Styler here, since we want to get people away from it as much as possible. We might even get rid of it completely at some point in the future. I would not mention it here.

Reply:
Since you've added the restriction to the docstring recently, I agree we can get rid of it here. I wanted to make sure that limitation was clear.